src/resources/SecurityCache/SecurityCache.ts

```diff
@@ -16,17 +18,35 @@ export default class SecurityCache extends Ressource {
     static providersUrl = `/${SecurityCache.baseUrl}/securityproviders`;
 
     listMembers(providerId: string, options?: SecurityCacheListOptions) {
-        return this.api.get<PageModel<DetailedSecurityCacheMemberModel[]>>(
```

Comment:
listing does not returns an array?

Reply:
https://github.com/coveo/platform-client/pull/203/files#diff-686dee64c39bb5a5c90dfbf8eda0629b09926fad94b4ffbd81c79ba6557e2fb7R131
I see

Reply:
The `PageModel` type already defines the `items` as an array of `T`.

![image](https://user-images.githubusercontent.com/35579930/100109739-23e31e80-2e3a-11eb-8b39-ca58aa30990c.png)